contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -194,7 +194,13 @@ contract VaultHub is PausableUntilWithRoles {
         // reserveRatio = TOTAL_BASIS_POINTS - maxMintableRatio
         // X = (mintedStETH * TOTAL_BASIS_POINTS - vault.valuation() * maxMintableRatio) / reserveRatio
 
-        return (mintedStETH * TOTAL_BASIS_POINTS - valuation * maxMintableRatio) / reserveRatioBP;
+        // Protection against underflow in division
+        if (!isHealthy && mintedStETH * TOTAL_BASIS_POINTS > valuation * maxMintableRatio) {
+            return (mintedStETH * TOTAL_BASIS_POINTS - valuation * maxMintableRatio) / reserveRatioBP;
+        }
+
+        // return MAX_UINT_256
+        return type(uint256).max;
```

Comment:
This is not reachable, because `!isHealthy` is the same with `mintedStETH * TOTAL_BASIS_POINTS > valuation * maxMintableRatio` basically.